### PR TITLE
Add song toolbar, provide ability to control listening and see music's info

### PR DESCRIPTION
**Pull request by Slawisch**

### CODE REVIEW

File: frontend/src/app/components/shared/song-toolbar/song-toolbar.component.ts [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  ngOnInit(): void {
+    this.playPauseButton = <HTMLButtonElement>document.getElementById('playbutton');
+    this.seekSlider = <HTMLInputElement>document.getElementById('seek-slider');
+    this.volumeSlider = <HTMLInputElement>document.getElementById('volume-slider');
+    this.currentTimeContainer = document.getElementById('current-time');
+    this.durationContainer = document.getElementById('duration');
+    this.audio = document.querySelector('audio');
+  }
+
+  UpdateSong = (songInfo: SongInfo) => {
```
Comment (NikitaKrasnovSW) [inline frontend/src/app/components/shared/song-toolbar/song-toolbar.component.ts]: Use camel case

File: frontend/src/app/components/shared/song-toolbar/song-toolbar.component.ts [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    else {
+      this.isMuted = true;
+      this.volumeSlider!.value = '0';
+      this.audio!.volume = 0;
+    }
+  };
+
+  toggleShuffle = () => {
+    this.isShuffling = !this.isShuffling;
+    (<HTMLButtonElement>document.getElementById('shufflebutton')).classList.toggle('active');
```
Comment (NikitaKrasnovSW) [inline frontend/src/app/components/shared/song-toolbar/song-toolbar.component.ts]: Use angular for toggling classes

**Review by NikitaKrasnovSW — CHANGES_REQUESTED** [review]

File: frontend/src/app/components/shared/song-toolbar/song-toolbar.component.ts (hunk 2 above)
Comment (Slawisch) [reply frontend/src/app/components/shared/song-toolbar/song-toolbar.component.ts]: Like here? 
https://stackoverflow.com/questions/44535515/angular-ngclass-and-click-event-for-toggling-class

Comment (NikitaKrasnovSW) [reply frontend/src/app/components/shared/song-toolbar/song-toolbar.component.ts]: Yep

**Review by NikitaKrasnovSW — APPROVED** [review]

**Pull request merged**